Bottom: fafe361d181d51f2251d59b28190705f896f2ae5
Top:    9631c573aa282b14a2eafa8987916d21989da92b
Author: Mark de Wever <koraq@xs4all.nl>
Date:   2021-08-12 21:07:41 +0200

Add a coverage validation script.


---

diff --git a/utils/check_coverage.py b/utils/check_coverage.py
new file mode 100755
index 0000000..63f77b6
--- /dev/null
+++ b/utils/check_coverage.py
@@ -0,0 +1,2 @@
+#!/usr/bin/env python3
+
